src/Orleans.Clustering.Cassandra/Membership/CassandraMembershipTable.cs

```diff
@@ -118,6 +118,31 @@ public async Task DeleteMembershipTableEntries(string clusterId)
             }
         }
 
+        public async Task CleanupDefunctSiloEntries(DateTimeOffset beforeDate)
```

Comment:
Implement this like in https://github.com/OrleansContrib/Orleans.Providers.MongoDB/blob/master/Orleans.Providers.MongoDB/Membership/Store/MongoMembershipCollection.cs#L95

Reply:
Method has been introduced in https://github.com/dotnet/orleans/blob/v2.3.0/src/Orleans.Core/SystemTargetInterfaces/IMembershipTable.cs#L30